pkg/plugins/plugins.go

```diff
@@ -346,19 +346,32 @@ func (s *PluginScanner) loadPlugin(pluginJSONFilePath string) error {
 	defer reader.Close()
 
 	jsonParser := json.NewDecoder(reader)
-	pluginCommon := PluginBase{}
-	if err := jsonParser.Decode(&pluginCommon); err != nil {
+	plugin := PluginBase{}
+	if err := jsonParser.Decode(&plugin); err != nil {
 		return err
 	}
 
-	if pluginCommon.Id == "" || pluginCommon.Type == "" {
+	if plugin.Id == "" || plugin.Type == "" {
 		return errors.New("did not find type or id properties in plugin.json")
 	}
 
-	pluginCommon.PluginDir = filepath.Dir(pluginJSONFilePath)
-	pluginCommon.Signature = getPluginSignatureState(s.log, &pluginCommon)
+	plugin.PluginDir = filepath.Dir(pluginJSONFilePath)
+	plugin.Files, err = collectPluginFilesWithin(plugin.PluginDir)
+	if err != nil {
+		s.log.Debug("Could not collect plugin file information in directory", "pluginID", plugin.Id, "dir", plugin.PluginDir)
```

Comment:
Why Debug level, as opposed to Warn considering it's an error?

Reply:
Just because the error will get collected and logged as warns anyway (same as below) - but can change this as it can provide more useful context as warn